From: Benjamin Drung <benjamin.drung@canonical.com>
Date: Wed, 13 Aug 2025 12:15:22 +0200
Subject: fix(Makefile): exclude hidden directories from shellcheck

`shfmt -f .` also searches in hidden directories (like `.git` or
`.pc`). The `.pc` is used on Debian for patch tracking and should not be
searched for files.

So ignore the top-level hidden directories from shellcheck.

Forwarded: https://github.com/dracut-ng/dracut-ng/pull/1568
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 73534c2..020d11d 100644
--- a/Makefile
+++ b/Makefile
@@ -317,7 +317,7 @@ syncheck:
 	done;exit $$ret
 ifeq ($(HAVE_SHELLCHECK),yes)
 ifeq ($(HAVE_SHFMT),yes)
-	shellcheck $$(shfmt -f .)
+	shellcheck $$(shfmt -f *)
 else
 	find . -name '*.sh' -print0 | xargs -0 shellcheck
 endif
